Tasks/VsTest/typings/vsts-task-lib.d.ts

```diff
@@ -1,12 +1,12 @@
 declare module 'vsts-task-lib/taskcommand' {
 	export class TaskCommand {
-	    constructor(command: any, properties: any, message: any);
-	    command: string;
-	    message: string;
-	    properties: {
-	        [key: string]: string;
-	    };
-	    toString(): string;
+		constructor(command: any, properties: any, message: any);
```

Comment:
correct.  You never want to check in the vsts-task-lib.  The task lib carries it with the lib which fixed a bunch of problems with them staying in sync.  I worked with the ts team and that's what they recommended.